### PR TITLE
Fix build deps, update docs

### PR DESCRIPTION
**Pull request by genebean**
- added `g++` and `yaml-dev` packages to resolve issues during python's build phase
- added OS package updating
- added updating of `pip`
- added updating of `setuptools` and associated packages per Python doc's page
- change owner to myself since Daniel is no longer at Puppet

### CODE REVIEW

**Comment by genebean** [comment]
v 0.9.3 is on PCR and based off of this PR @heathseals

File: setup.py [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
 setuptools.setup(
     name = "p9-admin",
-    version = "0.9.2",
+    version = "0.9.3",
 
     description = "Administrative tools for Platform9",
-    author = "Daniel Parks",
-    author_email = "daniel.parks@puppet.com",
+    author = "Gene Liverman",
+    author_email = "gene.liverman@puppet.com",
```
Comment (heathseals) [inline setup.py]: Should we update the author to be team-infracore?

Comment (genebean) [reply setup.py]: Possibly but it will need to be some other address as we don't let the world hit that one.

Comment (heathseals) [reply setup.py]: got it.  i just now realized this repo is public. : ) no reason to block on this.

**Pull request merged**